node.js

```diff
@@ -1,15 +1,21 @@
 "use strict";
 
+const nodePlugin = require("eslint-plugin-n");
+const baseConfig = require("./common.js");
+
+const cjsConfig = nodePlugin.configs["flat/recommended-script"];
+
 module.exports = {
-  extends: ["./index.js", "plugin:n/recommended"],
-  env: {
-    node: true,
-  },
-  parserOptions: {
+  ...baseConfig,
+  ...cjsConfig,
+  name: "cesium/node",
+  languageOptions: {
+    ...cjsConfig.languageOptions,
     ecmaVersion: 2023,
   },
-  plugins: ["n"],
```

Comment:
I'm trying to convert a simple application to use this, and it's complaining that the `n` plugin isn't installed. Shouldn't this:

```
plugins: {
  "n": n
}
```

where n is the required in plugin?

Reply:
I think you are also missing actually activating the plug-in, via `...eslintPluginN.configs["flat/recommended-script"]` at the top.

Reply:
Nevermind the second part, I see you called that `cjsConfig` and I missed it.

Reply:
I set up `eslint-plugin-n` as described in their docs: https://www.npmjs.com/package/eslint-plugin-n

I can add a line to the readme to recommend installing `eslint-plugin-n`.

Reply:
It may not be needed, but it was giving me an error. How confident are you in all of these configs being set up properly?

Reply:
Pretty confident at this point. I have it working over in CesiumJS. Why do you ask?

Reply:
Because I have no idea what I'm doing and was using some of your code as a reference. Once I get my one small project working, I'll be confident enough to merge this.